cron/centAcl.php

```diff
@@ -110,9 +111,29 @@
         programExit($errorMessage);
     }
 
-    $resourceCache = array();
+    /**
+     * Sync ACL with LDAP's contactgroup
+     * If the LDAP is enable and the last check is greater than the update period
```

Comment:
Just to be niggling but there is small English error on the comment. "enable" => "enabled"